src/wire/mod.rs

```diff
@@ -123,6 +125,11 @@ pub use self::ipv6::{Address as Ipv6Address,
                      Cidr as Ipv6Cidr,
                      MIN_MTU as IPV6_MIN_MTU};
 
+#[cfg(feature = "proto-ipv6")]
+pub use self::ipv6option::{Packet as Ipv6OptionPacket,
+                        Repr as Ipv6OptionRepr,
```

Comment:
nit: indentation

Reply:
hmm, looks like my vim settings are masking some indentation problems.

Reply:
n/m, that was simply my oversight.

Reply:
+1 for vim